app/controllers/api/v1/statistics_controller.rb

```diff
@@ -26,13 +17,28 @@ def create
 
   private
 
+  def make_stat(tones)
+    @statistic = Statistic.new
+    authorize @statistic
+    tweet_scores = tones.map { |tone| tone["score"] }
```

Comment:
Did you test this? Just double checking, because when I tried "score" with quotes, it didn't work ... hmmm ...